extra-files/galaxy-kickstart/welcome.html

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+    <title>Homepage</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link rel="stylesheet" href="style/base.css" type="text/css" />
+</head>
+<body>
+    <div class="document">
+        <p align="center">
+            <img src="images/galaxy-kickstart_logo.png" width="400" />
+        </p>
+    </div>
+    <div class="container">
```

Comment:
Something like this:
https://github.com/galaxyproject/usegalaxy-playbook/blob/master/templates/galaxy/common/static/welcome.html.j2
![screen shot 2016-05-30 at 18 43 32](https://cloud.githubusercontent.com/assets/6804901/15654658/566fcc1e-2696-11e6-821b-2ff68ba14797.png)